hazelcast/src/hazelcast/client/spi/LifecycleService.cpp

```diff
@@ -89,8 +97,12 @@ namespace hazelcast {
                 util::ILogger &logger = util::ILogger::getLogger();
                 switch (lifecycleEvent.getState()) {
                     case LifecycleEvent::STARTING :
-                        logger.info("LifecycleService::LifecycleEvent STARTING");
+                    {
+                        char msg[100];
+                        util::snprintf(msg, 100, "(%s:%s) LifecycleService::LifecycleEvent STARTING", HAZELCAST_GIT_COMMIT_DATE, HAZELCAST_GIT_COMMIT_ID);
```

Comment:
Is the date format as YYYYMMDD? PRD wants it this way. You can just convert it. See https://github.com/hazelcast/hazelcast-cpp-client/pull/90/files#diff-25e2207206ddcc5a7f4063ce1e5c2853R94